content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-state-2.md

```diff
@@ -42,7 +42,7 @@ Include these variables in the `BoardingTicket` data class:
 * `description`: Trip information. Use type `String`.
 * `marsExpress`: The space travel company issuing the ticket. Use type `Party`.
 * `owner`: The party exchanging the voucher for the ticket. Use type `Party`.
-* `daysUntilLaunch`: The number of days until the launch date. Use type `int`.
+* `launchDate`: The number of days until the launch date. Use type `LocalDate`.
```

Comment:
launch date isn't the number of days till the launch, its a date :)

Reply:
Good catch!